src/mongocrypt-kms-ctx.c

```diff
@@ -885,6 +991,10 @@ _mongocrypt_kms_ctx_result (mongocrypt_kms_ctx_t *kms,
 {
    mongocrypt_status_t *status;
 
+   if (!kms || !out) {
```

Comment:
Suggest converting into not-null assertions.

Reply:
Changed as suggested.  Will show up in next commit.